internal/misc/sanitizer.go

```diff
@@ -0,0 +1,9 @@
+package misc
+
+import "strings"
+
+// SanitizeString sanitizes string before using it in logs. Required
+// for data from the user input: request body, headers, etc.
+func SanitizeString(s string) string {
+	return strings.Replace(s, "\n", "", -1)
```

Comment:
Code scanning alerts suggested empty string there. I guess it doesn't matter for us because strings with new lines are malicious, they don't have to be pretty.